### DISCUSSION TITLE
how to build a atlas  cluster?

### DISCUSSION THREAD

**Discussion by tyoldwu** [aws/aws-sdk-java#3019] [General]
1. In memory mode，how to generate a snapshot of data for easy recovery after an instance is suspended？
2. How to implement multiple replicas？
3. How to implement multiple shards？
thank you

**Comment by debora-ito** [comment]
Hey @tyoldwu these are service specific questions, not questions about the Java SDK, which is the focus of this channel.

Assuming you are asking about MongoDB Atlas, the AWS and MongoDB doc page is a good place to start - https://aws.amazon.com/partners/mongodb/
